scripts/load_reference_data.sh

```diff
@@ -11,16 +11,27 @@ check_exists() {
     fi
 }
 
+safe_execute() {
```

Comment:
This is cute.  :)  Does it work locally?  If so looks good.

Reply:
Yup, works locally. I got lazy and didn't want to write fail code everywhere. :)